opencraft/settings.py

```diff
@@ -295,6 +295,10 @@
 # for provisioning Open edX app servers.
 DEFAULT_CONFIGURATION_VERSION = env('DEFAULT_CONFIGURATION_VERSION', default=DEFAULT_OPENEDX_RELEASE)
 
+# Git tag for latest Open edX release. Used as a default refspec for
+# configuration, edx-platform, forum, notifier, xqueue, and certs when creating production instances.
+LATEST_OPENEDX_RELEASE = env('LATEST_OPENEDX_RELEASE', default='named-release/dogwood.rc')
```

Comment:
Should we use the branch or the tag, at the end? Also, I don't see a dogwood branch on https://github.com/open-craft/edx-platform yet - what branch/ref will we be using on our fork?

Also, the latest Open edX release isn't really something which is configuration dependent - maybe this should be called `OPENEDX_RELEASE_STABLE_REF` or something like that, to reflect that it's more the release branch to use, rather than necessarily the last one edX released.